New package: RecursiveCausalDiscovery v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: RecursiveCausalDiscovery
- Repository: https://github.com/sepehr78/RecursiveCausalDiscovery.jl
- Created by: @sepehr78
- Version: v0.1.0
- Commit: 8e2f615e02fda7304808acc4de1a09c450412215
- Reviewed by: @sepehr78
- Reference: https://github.com/sepehr78/RecursiveCausalDiscovery.jl/commit/8e2f615e02fda7304808acc4de1a09c450412215#commitcomment-144078233
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Initial release! 🎉
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd626c163fea02311478ee8efe3afdeaa32d66894e7cf51d7def5810ed2b407af722561b8fde91b155696fe54e3aa8c7996d04e8208720f7bb4d00034e4373306f87453785ec0aa40963a56e8c858a55ba4f71adb549fa469e15b80949f2e1f1cce5c563200d21c8405c724207f2b3120324f8a8c2a28fa6a6e74cc76dad6e3c50deea49e3befb4acae439b8e03f475c0ca24259562c5ba9649583d7b0c5e57674232ead18679279dc594a6b9d33a160ac5b392308a2a8d1d0d82324809c2210eb5 -->

gdalle commented:
Congrats @sepehr78 on the new package!
Can I suggest that you add a minimal README and documentation to your package, as part of the [recommended best practices](https://github.com/JuliaRegistries/General?tab=readme-ov-file#should-i-register-my-package-now) for registration?

sepehr78 commented:
> Congrats @sepehr78 on the new package! Can I suggest that you add a minimal README and documentation to your package, as part of the [recommended best practices](https://github.com/JuliaRegistries/General?tab=readme-ov-file#should-i-register-my-package-now) for registration?

Done!

JuliaRegistrator committed:
New package: RecursiveCausalDiscovery v0.1.0

UUID: 96483cbc-aa6d-4394-9266-bb69d9075284
Repo: https://github.com/sepehr78/RecursiveCausalDiscovery.jl.git
Tree: 04b95d0ca00b804edd6f78dbe408ded3a3862280

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
